### PR TITLE
LT-1215-Validation-missing

### PR DESCRIPTION
**Pull request by 5pence**

### CODE REVIEW

**Comment by phlptt** [comment]
Hi @5pence could you update the e2e tests for this as well please? There's a test for it already for it with a few steps conveniently commented out. It should just be the case of uncommenting lines 111, 112 and 113 within /ui_automation_tests/features/submit_application.feature and then running it. give me a shout if you need a hand. cheers

File: drafts/serializers.py [hunk 1]
```diff
@@ -38,6 +39,9 @@ class DraftCreateSerializer(DraftBaseSerializer):
     export_type = serializers.ChoiceField(choices=ApplicationExportType.choices, error_messages={
                                               'required': 'Select if you want to apply for a temporary or permanent '
                                                           'licence.'})
+    have_you_been_informed = serializers.ChoiceField(choices=ApplicationExportLicenceOfficialType.choices,
+                                                     error_messages={'required': 'Select if you you been told that you '
```
[the remark this answers is not in the document]
Comment (antonhughes) [reply drafts/serializers.py]: this is inconsistant with all other serializers, is this a recent this that we need to change?

[the remark this answers is not in the document]
Comment (lukesa) [reply drafts/serializers.py]: yes, good example:
https://github.com/uktrade/lite-api/pull/61/files/638d33d23c215febfb36cb97d4f83c57f157fe8c..3059ad810f626f141a186c731d522650edcc8940

[the remark this answers is not in the document]
Comment (antonhughes) [reply drafts/serializers.py]: done

File: drafts/serializers.py [hunk 2]
```diff
@@ -58,6 +63,9 @@ class DraftUpdateSerializer(DraftBaseSerializer):
...
```
[the remark this answers is not in the document]
Comment (antonhughes) [reply drafts/serializers.py]: done

**Pull request merged**